lib/screens/login/view/login_screen.dart

```diff
@@ -203,6 +205,10 @@ class LoginScreenState extends State<LoginScreen>
     ];
   }
 
+  bool showBiometrics() {
+    return (!globals.testingActive && Platform.isIOS) || Platform.isAndroid;
```

Comment:
We should disable biometrics when running e2e tests and later add a unit test for the biometrics widget.
`Platform.isIOS` and `Platform.isAndroid` can have support biometrics or not depending if the app is being ran on a physical device or in an emulator.

maybe update this bit to ->
```
bool showBiometrics() {
    return !globals.testingActive;
```

Reply:
Biometrics can only be shown in IOS and Android, we still need that check. It does not matter if IOS is running on a physical device or an emulator, e2e testing does not detect the native popups. Is Android only detecting native pop ups on physical device?